### DISCUSSION TITLE
全局修改字体大小问题

### DISCUSSION THREAD

**Discussion by pengyinghao** [vueComponent/ant-design-vue#5775] [General]
环境是vite+vue3+ant design vue 3.29
目前所有组件默认的14px，但是我用了css引入的方式去动态更换主题，怎么去修改字体大小呢。 
ConfigProvider.config({}) 这个还不支持修改字体，后续会支持吗

**Comment by doommm** [comment]
替换 less 字体变量？好像是 `@font-size-base` 这个，可以去源码里对应变量的位置查一下

**Comment by pengyinghao** [comment]
问题就是我没用less，引入的css，现在又改为了less来做配置。
